client/components/login.jsx

```diff
@@ -0,0 +1,80 @@
+/* eslint-disable no-unused-vars */
+import React from 'react';
+
+class Login extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      email: 'chris@gmail.com',
+      password: '12345'
+    };
+    this.handleSubmit = this.handleSubmit.bind(this);
+    this.handleChange = this.handleChange.bind(this);
+    this.handleChange2 = this.handleChange2.bind(this);
+    this.registerButton = this.registerButton.bind(this);
+  }
+
+  handleChange(event) {
+    this.setState({ email: event.target.value });
+  }
+
+  handleChange2(event) {
+    this.setState({ password: event.target.value });
+  }
```

Comment:
You should only have 1 `handleChange` method, imagine if you had 10 inputs, you don't want to have to have separate handle change methods for each one. If you add a `name` attribute to each input set to the correct property name in state you can use that information to make a dynamic handle change method:

```JAVASCRIPT
handleChange(event) {
  const name = event.target.name;
  const value = event.target.value;
  
  this.setState({ [name]: value });
}
```